js/gateio.js

```diff
@@ -622,7 +622,14 @@ module.exports = class gateio extends Exchange {
                 for (let i = 0; i < response.length; i++) {
                     const market = response[i];
                     const id = this.safeString (market, 'name');
-                    const [ baseId, quoteId, date ] = id.split ('_');
+                    const ids = id.split ('_');
+                    const baseId = ids[0];
+                    const quoteId = ids[1];
+                    let date = undefined;
+                    const idLength = ids.length.toString ();
+                    if (Precise.stringGt (idLength, '2')) {
```

Comment:
we don't need to use `Precise.stringGt` in this instance since we are just comparing ints and not storing the value